manager/integration/tests/test_engine_upgrade.py

```diff
@@ -111,6 +112,16 @@ def test_engine_offline_upgrade(clients, volume_name):  # NOQA
     assert volume["engineImage"] == original_engine_image
     assert volume["currentImage"] == original_engine_image
 
+    # Before our upgrade, write data to the volume first.
+    test_hostid = get_self_host_id()
```

Comment:
You can just use `host_id = get_self_host_id()` then you don't need to change some codes below.